Extension/package.json

```diff
@@ -42,6 +42,168 @@
   ],
   "main": "./dist/main",
   "contributes": {
+    "walkthroughs": [
+      {
+        "id": "cppWelcome",
+        "title": "Get Started with C++ Development",
```

Comment:
Should the root level "question" be something like "Do you want to do C or C++ development?" or is that better for a V2 or not possible yet?

Reply:
In blog posts and conference talks we just say C++, and the audience seems to understand that it encompasses both C++ and C. I want to be consistent with our recent branding of calling it "the C++ extension," but I think this would be a good thing to experiment with!